lib/server/model.js

```diff
@@ -167,11 +181,35 @@ class RangerModel {
     return attrs;
   }
 
+  _fbAttrs(query) {
+    assert(query);
+    assert(RangerModel.hasReference(query),
+      'database does not hold location of reference .fa file');
+    // '-c' option allows freebayes to read from stdin ie. from pipe
+    let attrs = ['-c', '-f', query.reference];
+    if (query.region) {
+      attrs.push('-r', query.region);
+    }
+    LOGGER.debug('freebayes attrs: ' + attrs);
+    return attrs;
+  }
+
+  _scheduleVCF(query, processes) {
+    const freebayes = child.spawn(FREEBAYES_COMMAND, this._fbAttrs(query));
+    freebayes.title = 'freebayes';
+    processes.push(freebayes);
+    return processes;
```

Comment:
The processes array is an object and is passed by reference. Therefore, no need to return it.  Example:
> var b = (arr) => { arr.push(3);};
undefined
> let a = [1, 2];
undefined
> b(a);
undefined
> console.log(a);
[ 1, 2, 3 ]